app/views/session_timeout/_ping.js.erb

```diff
@@ -1,6 +1,7 @@
 var frequency = <%= frequency %> * 1000;
 var warning = <%= warning %> * 1000;
 var start = <%= start %> * 1000;
+var timeoutPath = <%= timeout_path.to_json %>;
```

Comment:
I guess `to_json` doesn't quite work, since it escapes the double-quote as `&quot;` and is logging an error. Your original may be the way to go for now. Maybe [`j` (`escape_javascript`)](https://api.rubyonrails.org/classes/ActionView/Helpers/JavaScriptHelper.html#method-i-j)?